lib/WorkflowCutandrun.groovy

```diff
@@ -18,7 +18,7 @@ class WorkflowCutandrun {
             System.exit(1)
         }
 
-        if (!params.spikein_fasta) {
+        if (params.normalisation_mode == "Spikein" && !params.spikein_fasta) {
```

Comment:
What happens if `normalisation_mode` is given as "spikein"

Reply:
It will error but gracefully saying it doesn't recognise the param

```
ERROR: Validation of pipeline parameters failed!


* --normalisation_mode: 'TEST' is not a valid choice (Available choices: Spikein, RPKM, CPM, BPM, None)
```

Reply:
Ok, that's fine!